psutil/net.go

```diff
@@ -102,7 +121,28 @@ func netIOCounters(nss []core.Namespace) ([]plugin.MetricType, error) {
 				}
 				results = append(results, metric)
 			}
-		} else {
+		case "netfilter":
+			if ns[4].Value == "conntrackcount" {
+				metric := plugin.MetricType{
+					Namespace_: ns,
+					Data_:      conntrack[0].ConnTrackCount,
+					Timestamp_: time.Now(),
+					Unit_:      netConntrackCounterLabels[metricName].unit,
+				}
+				results = append(results, metric)
+			}
+
+			if ns[4].Value == "conntrackmax" {
+				metric := plugin.MetricType{
+					Namespace_: ns,
+					Data_:      conntrack[0].ConnTrackMax,
+					Timestamp_: time.Now(),
+					Unit_:      netConntrackCounterLabels[metricName].unit,
+				}
```

Comment:
it seems that you can combine two if statements together as data is the difference.